p2p/discovery_test.go

```diff
@@ -0,0 +1,31 @@
+package p2p
+
+import (
+	"context"
+	"testing"
+
+	"github.com/libp2p/go-libp2p"
+	"github.com/libp2p/go-libp2p/p2p/muxer/yamux"
+	"github.com/stretchr/testify/require"
+)
+
+func TestDiscoverPeers(t *testing.T) {
+	require := require.New(t)
+
+	ctx := context.Background()
+	bootNodeMultiaddr := "/dns4/bootnode-0.testnet.iotex.one/tcp/4689/ipfs/12D3KooWFnaTYuLo8Mkbm3wzaWHtUuaxBRe24Uiopu15Wr5EhD3o"
```

Comment:
maybe we should move this to smoke test
for unit test should not depend out resource